### DISCUSSION TITLE
When will it be available for the official 8.0 version?

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [CommunityToolkit/Windows#102] [General]
This community toolkit is a great project, and I'm using it in my Windows App SDK project, and I'd like to ask when I'll be able to use the official 8.0 version.

**Comment by kmgallahan** [comment]
You can track required work items here:

https://github.com/orgs/CommunityToolkit/projects/7

I imagine the target for a preview release is Q3/Q4 (potentially coinciding with .NET 8 release), but realistically it is a case of "it'll be ready when it's ready".

**Reply by michael-hawker** [reply]
Yup, we've been using that project to track items. Our goal is to actually get something out a bit sooner, then we'll probably ship a smaller update release later in the year once the Windows App SDK 1.4 release ships. Many things are working and updated, so we'd rather release what we have and move forward in phases of continuing to bring other things forward and adding new components as we go along in this new space.
